src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const newSelectedTabId = selectedTabId
+    && tabs.some(tab => tab.id === selectedTabId) ? selectedTabId : tabs[0].id;
```

Comment:
this condition you have in state `[selectedTabId, setSelectedTabId] = useState(tabs[0].id);`